drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -620,8 +620,14 @@ def show_answer(self, data, suffix=''):
 
         answer = self._get_correct_state()
 
-        answer['explanation'] = self.data.get('explanation', '').strip()
-
+        explanation = self.data.get('explanation', '').strip()
```

Comment:
@abodacs 
```suggestion
        explanation = _clean_data(self.data.get('explanation', '').strip())
```

Reply:
I revered adding  ```_clean_data``` suggestion as it removed HTML tags

Reply:
But it was working fine when I was testing it the other day. I will check its impact.